scalyr_agent/configuration.py

```diff
@@ -1130,7 +1130,7 @@ def __verify_main_config_and_apply_defaults(self, config, file_path, apply_defau
         # Optional (defaults to False). If true, will use the controlled warmer strategy to warm the pod
         # cache.  This only applies when docker is being used as the container system
         self.__verify_or_set_optional_bool(
-            config, 'k8s_use_controlled_warmer', False, description, apply_defaults, env_aware=True
+            config, 'k8s_use_controlled_warmer', True, description, apply_defaults, env_aware=True
```

Comment:
Partly to protect against the user forgetting to set config variable.
Partly because I feel this is stable/tested enough to become default True.

Reply:
I agree that we should probably be looking to make the controlled warmer the default, however, we also need to be mindful of startup times for normal/medium sized clusters which seem to be able to cope with the existing loads without any issues.

This means tweaking the defaults such that most small/medium clusters won't notice any limits on rates (because actual rps will always be less that target rps).

Does the current default of 40rps initially seem to work well on your test clusters.

Also note, the comment *above* this still mentions the default value is `False`, which is a contradiction of the code.

Reply:
I'll fix the comment. How bout we wait until after Monday to discuss changing default values and removing that json logging todo. (The json logging is still useful for my debugging as I'm bringing test clusters up/down a lot).  

These changes really pertain to when we merge `option-b/ratelimit` back into master right? (ie they can safely stay in the `option-b/ratelimit` branch.

Reply:
> (ie they can safely stay in the option-b/master branch.

They can, we just need to be mindful to go through all TODOs before merging.